guides/installing-python.md

```diff
@@ -0,0 +1,147 @@
+```
+
+**Note:** pyenv-virtualenvwrapper is not strictly required, depending on how you wish to manage your virtualenvs. See the [notes in the pyenv-virtualewrapper README](https://github.com/pyenv/pyenv-virtualenvwrapper#pyenv-virtualenvwrapper). Some of our documentation assumes `virtualenvwrapper` exists, so this guide installs it. You will see a deprecation warning when using `mkvirtualenv`:
+
+```
+WARNING: the pyenv script is deprecated in favour of `python3.6 -m venv`
+```
+
+Until we switch to using Python 3 and the `venv` module full-time, this can be safely ignored.
+
+## Configuring pyenv
+
+Per the [pyenv documentation](https://github.com/pyenv/pyenv#basic-github-checkout), the follow environment variables need to be added to your shell dotfiles (`.bashrc` for bash users, `.zshrc` for zsh users).
+
+```shell
+# pyenv
+export PYENV_ROOT="$HOME/.pyenv"
+
+# pyenv-virtualenvwrapper
+export PYENV_VIRTUALENVWRAPPER_PREFER_PYVENV="true"
+export WORKON_HOME=$HOME/.virtualenvs
+
+# Ensure this placed toward the end of the file
+eval "$(pyenv init -)"
+
+# Load virtualenvwrapper
+pyenv virtualenvwrapper_lazy
+```
+
+Restart your shell.
+
+## Installing Python
+
+We have projects that require both Python 2.7 and Python 3.6:
+
+```shell
+pyenv install 3.6.8
+pyenv install 2.7.15
+```
+
+With these two versions of Python installed, you 
+
+## Setting the global Python versions
+
+By default with pyenv when you invoke `python`, you will be using the system-installed Python. To set the [global Python version to the versions installed above](https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-global):
+
+```shell
+pyenv global 3.6.8 2.7.15
+```
+
+This will make `python` version 3.6.8, and `python2` will be 2.7.15. This is our recommended configuration.
+
+
+## Using local Python versions
+
+pyenv also allows you to set a local Python version (the version of Python that is available in the current directory):
+
+```shell
+pyenv local 2.7.15
+```
+
+This will create a `.python-version` file in the current directory. When you're in that directory or below it, pyenv will use the version you've specified here as `python`. 
+
+**Note:** When you create a virtualenv with `mkvirtualenv` with a local Python version set, it will default to that local Python version, and not the global version. You can still specify `-p` argument with the specific Python you wish to use.
+
+## Upgrading Python versions
+
+To change the local Python version to the new version, for example, 3.7.2, follow the same approach you took to setting it the first time:
+
+```shell
+pyenv local 3.7.2
+```
+
+And the local `.python-version` file will contain `3.7.2`.
+
+This does not work for virtualenvs, however. If you install a new version of Python, for example, 3.7.2, you must recreate the virtualenv with the new version of Python. The virtualenv will have be recreated (`mkvirtualenv`) with the new version of Python either set locally, globally, or specified with `-p`.
```

Comment:
What does that look like as a command? I don't quite understand what this is trying to instruct.

Reply:
Documented in 33495a5